system/indy-node-tests/test_upgrade.py

```diff
@@ -111,19 +112,17 @@
     # print(res)
     # assert res['op'] == 'REPLY'
 
-    await asyncio.sleep(300)
+    await asyncio.sleep(7*5*60)
```

Comment:
Was it intended to sleep for 35 minutes in this test?

Reply:
In case of not forced upgrade - yes, since we have at least 5 minutes delay between each node's upgrade and we have 7 nodes for docker setup.